### DISCUSSION TITLE
Internal mode

### DISCUSSION THREAD

**Discussion by joachimBrindeau** [kimai/kimai#2750] [General]
Hello,

It would be great to have an "internal mode" for companies only looking to track internal projects.
A simple switch to disable everything related to clients, billing and invoices.

Looking forward to know what you think!
Cheers

**Comment by kevinpapst** [comment]
Kimai is primarily a project-time tracking tool, so disabling features like clients doesn't make sense.

If you have only one customer (your own company), then it shouldn't appear in most forms. 

Invoicing can be disabled, like many other features, by removing the role permissions. 

If your company wants to sponsor certain other aspects, that might make it more usable for you and everyone else tracking internal times, I'd be happy to discuss that.

**Reply by joachimBrindeau** [reply]
Thanks a lot for the answer.

I was just asking because my background is in biotech and now I work with labs that often look for a time-tracking tool.
I'm currently browsing for minimalistic solutions that would allow them to track the time on each project with no bloat (meaning nothing this customer segment doesn't need).
I'd be glad to talk once I have a better view of whether it's a real need or not.

Thanks again :)

**Reply by kevinpapst** [reply]
As written: many features can be deactivated by permissions.

The customers drop-down is removed if you only have one customer registered (you still need your own company as "parent" for all registered projects).

Even though the companies might think they want "a minimal solution" they often tend to increase their needs quickly after starting to use a time-tracking tool.

**Reply by joachimBrindeau** [reply]
Aaah sorry I didn't understand the permission thing at first.
It makes sense! Thank you!
